packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -33,227 +45,359 @@
-      @include bpk-body-longform;
+    @include bpk-breakpoint-tablet {
+      @include bpk-heading-1;
+
+      &:not(:last-child) {
+        margin-bottom: bpk-spacing-md();
+      }
     }
-  }
 
-  @include breakpoint-portrait-small {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+    @include bpk-breakpoint-small-tablet {
+      margin-bottom: bpk-spacing-md();
 
-      @include bpk-label-1;
+      @include bpk-heading-1;
     }
 
-    &__tagline {
+    @include bpk-breakpoint-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-label-1;
+      @include bpk-hero-5;
     }
 
-    &__headline {
+    @include bpk-breakpoint-small-mobile {
       margin-bottom: bpk-spacing-md();
 
       @include bpk-heading-2;
     }
+  }
+
+  &__subheading {
+    @include bpk-breakpoint-above-tablet {
+      @include bpk-body-default;
+    }
+
+    @include bpk-breakpoint-tablet {
+      @include bpk-body-default;
+    }
 
-    &__subheading {
+    @include bpk-breakpoint-small-tablet {
+      @include bpk-subheading;
+    }
+
+    @include bpk-breakpoint-mobile {
+      @include bpk-body-longform;
+    }
+
+    @include bpk-breakpoint-small-mobile {
       @include bpk-body-default;
     }
   }
 
-  &__container {
+  &__promo-content,
+  &__sponsor-content {
     display: flex;
     flex-direction: column;
 
-    @include breakpoint-landscape {
-      justify-content: center;
+    @include bpk-breakpoint-above-tablet {
+      width: 22.5rem;
 
       &--start {
         align-items: start;
+        text-align: start;
       }
 
       &--center {
         align-items: center;
+        text-align: center;
       }
 
       &--end {
-        align-items: end;
+        align-items: start;
+        text-align: start;
       }
     }
 
-    @include breakpoint-portrait {
-      justify-content: space-between;
+    @include bpk-breakpoint-tablet {
+      width: 20rem;
 
-      &--inverted {
-        flex-direction: column-reverse;
+      &--start {
+        align-items: start;
+        text-align: start;
+      }
+
+      &--center {
+        align-items: center;
+        text-align: center;
+      }
+
+      &--end {
+        align-items: start;
+        text-align: start;
       }
     }
 
-    @include breakpoint-landscape-desktop {
-      height: 25rem;
+    @include bpk-breakpoint-small-tablet {
+      width: initial;
+      align-items: initial;
+      text-align: initial;
+    }
 
-      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
-      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
+    @include bpk-breakpoint-mobile {
+      width: initial;
+      align-items: initial;
+      text-align: initial;
+    }
 
-      /* adjusting for 1.5rem different between margins */
-      &--center > * {
-        @include bpk-margin-trailing(bpk-spacing-lg());
-      }
+    @include bpk-breakpoint-small-mobile {
+      width: initial;
+      align-items: initial;
+      text-align: initial;
     }
+  }
```

Comment:
One thing it might be worth trying to clean this up (no idea if this will work in this case)

```
@include bpk-breakpoint-small-tablet,
@include bpk-breakpoint-mobile,
@include bpk-breakpoint-small-mobile {
 width: initial;
 align-items: initial;
 text-align: initial;
}
```

Reply:
Yeah, I tried this, but SCSS does not support it. We need to either write a complicated mixin with `if...else` or create a mixin for the 3 properties, then include it in each breakpoint. 😞

Reply:
Given that it is 3 properties, I thought writing another mixin isn't worth it.